rwengine/src/core/Logger.hpp

```diff
@@ -13,7 +13,7 @@
  */
 class Logger {
 public:
-    enum MessageSeverity { Verbose, Info, Warning, Error };
+    enum MessageSeverity { Verbose = 'V', Info = 'I', Warning = 'W', Error = 'E' };
```

Comment:
I'ld just use a constexpr char-array with an int enum used as indices.

Reply:
The MessageSeverity enum should be in severity order, rather than alphabetical order.